main.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+
+    std::cout << "JOE'S ACME SOFTWARE, INC. STOCK VALUES\n\n"; // just some header text
+
+    //stock purchase calculations
+    const int SHARES_BOUGHT = 1000;
+    const double PURCHASE_PRICE = 45.50;
+    double total_stock_price = SHARES_BOUGHT * PURCHASE_PRICE; // price of all stock
+    const double COMMISSION_RATE = .02;
+    double total_commission = total_stock_price * COMMISSION_RATE; // total commission broker will be paid
+    double purchase_grand_total = total_stock_price + total_commission; //total cost after commission is factored in
+
+    //print buying results
+    std::cout << "BUYING\n" << "====================================\n";
+    std::cout << std::setprecision(2) << std::fixed;
```

Comment:
I like that you left this decimal formatting on its own line. Makes it look very clean!